tests/atomic/main.yml

```diff
@@ -471,3 +471,46 @@
     - role: command
       cmd: atomic images update {{ local_name }}:latest
       output: "Writing manifest to image destination"
+
+
+- name: Atomic- Cleanup
+  hosts: all
+  become: true
+
+  tags:
+    - cleanup
+
+  tasks:
+    - name: Check if 'atomic-registries' was layered in
+      shell: >
+        rpm-ostree status --json |
+        docker run -i docker.io/miabbott/aht-tools
+        jq '.deployments[0]["base-commit-meta"]["rpmostree.packages"]'
+      register: ros
+
+    - name: Indicate 'atomic-registries' is layered
+      when: "'atomic-registries' in ros.stdout"
+      set_fact:
+        ar_is_layered: true
```

Comment:
This assumes deployment 0 is booted, which is likely the case.  Can we resuse the rpm_ostree_status role that sets the booted deployment, then check the layered packages?

```
import_role:
  name: rpm_ostree_status

when: "'atomic-registries' in ros_booted["base-commit-meta"]["rpmostree.packages"]
set_fact:
...
```

Reply:
We have high confidence about the state of the system, which is why I hard coded the first deployment in this case.

But I'll adapt it to check for 'booted' since you never know when things will go sideways.

Reply:
Knowing me, I would add a test before it and make it goes sideways :laughing: